cumulus_library/cli.py

```diff
@@ -2,8 +2,10 @@
 """Utility for building/retrieving data views in AWS Athena"""
 
 import argparse
+import json
 import os
 import sys
+import sysconfig
```

Comment:
lol always scary to see this import :smile:

Reply:
i'm open to suggestions, i'd love something higher level.

Reply:
Naw yeah, for the way you're doing this, this is appropriate. Just one of those imports that's like "we're about to get crazy, buckle up"